cloudformation/aws-parallelcluster.cfn.json

```diff
@@ -932,6 +937,24 @@
           ]
         }
       ]
+    },
+    "EnableDCV": {
+      "Fn::Equals": [
+        {
+          "Fn::Select": [
+            "0",
+            {
+              "Fn::Split": [
+                ",",
+                {
+                  "Ref": "DCVOptions"
+                }
+              ]
+            }
+          ]
+        },
+        "master"
```

Comment:
I'm almost leaning towards making this parameter a boolean, then defaulting it to false. If in the future we added the ability to have another master or enable DCV on the compute nodes, we could change this.